engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/sub/runtime/impl/JobResourceImpl.java

```diff
@@ -72,10 +73,13 @@ public String getStacktrace() {
   public void setJobRetries(RetriesDto dto) {
     try {
       ManagementService managementService = engine.getManagementService();
-      managementService.setJobRetries(dto.getRetries())
-        .jobId(jobId)
-        .dueDate(dto.getDueDate())
-        .execute();
+      SetJobRetriesBuilder builder = managementService
+          .setJobRetries(dto.getRetries())
+          .jobId(jobId);
+      if (dto.isDueDateSet()) {
+        builder.dueDate(dto.getDueDate());
```

Comment:
❓ Do we need to add this logic to the other endpoints as well?
* `JobDefinitionResourceImpl#setJobRetries`
* `ProcessInstanceRestServiceImpl#setRetriesByProcess`
* `ProcessInstanceRestServiceImpl#setRetriesByProcessHistoricQueryBased`
* `JobRestServiceImpl#setRetries`

Reply:
Yes, I forgot that. Sorry :(